pkg/cluster/install.go

```diff
@@ -411,11 +411,11 @@ func (m *manager) bootstrap() []steps.Step {
 
 	s = append(s,
 		steps.AuthorizationRetryingAction(m.fpAuthorizer, m.attachNSGs),
-		steps.Action(m.updateAPIIPEarly),
-		steps.Action(m.createOrUpdateRouterIPEarly),
-		steps.Action(m.ensureGatewayCreate),
-		steps.Action(m.createAPIServerPrivateEndpoint),
-		steps.Action(m.createCertificates),
+		steps.AuthorizationRetryingAction(m.fpAuthorizer, m.updateAPIIPEarly),
+		steps.AuthorizationRetryingAction(m.fpAuthorizer, m.createOrUpdateRouterIPEarly),
```

Comment:
Makes sense to me. I like the idea of using AuthorizationRetryingAction to subsequent steps as well, to avoid the potential issue of us hitting the auth failure later on down the line.

Reply:
Also, I confirmed that we've hit auth failures further down the line (during `m.createAPIServerPrivateEndpoint` specifically as well)